### PR TITLE
Export komodo release

### PR DESCRIPTION
**Pull request by joakim-hove**
Just a suggetsion - will support the komodo release as an environment variable.

### CODE REVIEW

**Comment by pgdr** [comment]
For which purpose, and will you export `stable` or (e.g.) `2018.01`?

Should users expect `/base/path/${KOMODO_RELEASE}/root/bin/exec` to work similarly as `exec`?

**Comment by joakim-hove** [comment]
This is just a suggestion - not important. I have just added `wlog` calls to the local script - to get a feeling of how succesfull the process of pushing people over to komodo is - curently the logging call is just:
```bash
wlog komodo start
```
but it would have been nice to have:
```bash
wlog komodo $KOMODO_RELEASE
```

The current approach will be based on the build argumet, so that will be e.g. `2018.02` and not `testing`.

> Should users expect /base/path/${KOMODO_RELEASE}/root/bin/exec to work similarly as exec?

I have mnot thought of that - but I guess so?!

File: enable.m4 [hunk 1]
```diff
@@ -1,2 +1,3 @@
 define(`komodo_prefix', `prefix')
 define(`komodo_pyver', `pyver')
+define(`komodo_release',`release')
```
Comment (pgdr) [inline enable.m4]: missing comma, but other than that, feel free to merge

**Comment by joakim-hove** [comment]
@pgdr : can you merge this - or give me the rights to do so?

**Comment by pgdr** [comment]
Sorry!  I've added the entire team as collaborators now … thought that was default.

**Pull request merged**